sepolia/config.yaml

```diff
@@ -0,0 +1,76 @@
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
+CONFIG_NAME: 'sepolia'
+
+# Genesis
+# ---------------------------------------------------------------
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 1300
+# Sunday, June 19, 2022 2:00:00 PM +UTC
+MIN_GENESIS_TIME: 1655647200
```

Comment:
these are out of date w/ your readme

Reply:
Genesis = min genesis time + genesis delay. Genesis delay is 86400 (1 day) so it should be correct